__tests__/nc_news.test.js

```diff
@@ -102,6 +108,27 @@ describe('GET /api/articles', () => {
 					});
 				});
 		});
+		test('GET:200 expects array of articles to only contain matching topics ', () => {
+			return request(app)
+				.get('/api/articles?topic=cats')
+				.then(({ body }) => {
+					const articles = body.articles;
+					expect(articles).toHaveLength(1);
+					articles.forEach((article) => {
+						expect(article.topic).toBe('cats');
+					});
+				});
+		});
+	});
+	describe('Endpoint error handling', () => {
+		test('GET:404 expects error when topic does not exist ', () => {
+			return request(app)
+				.get('/api/articles?topic=dogs')
+				.expect(404)
+				.then(({ body }) => {
+					expect(body.msg).toBe('topic does not exist');
+				});
+		});
```

Comment:
Good testing so far - I'd like to see one more 200 test for a topic that exists but doesn't have any articles associated with it (paper)